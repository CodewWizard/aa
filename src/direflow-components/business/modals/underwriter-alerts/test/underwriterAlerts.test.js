import React from "react";
import renderer from "react-test-renderer";
import UnderwriterAlerts from "../underwriterAlerts";
import TestRenderer from "react-test-renderer";

const commonArgs = {
  uwAlerts: true,
  setUWAlerts: () => {},
  summary: {
    Rules: {
      MatchingRules: [
        {
          Description: "Description1",
          Reason: "Reason1",
        },
        {
          Description: "Description2",
          Reason: "Reason2",
        },
      ],
    },
  },
  handleViewAlerts: () => {},
};

it("renders without crashing without props", () => {
  const testRenderer = TestRenderer.create(<UnderwriterAlerts />);
  expect(testRenderer.toJSON()).toMatchSnapshot();
  testRenderer.unmount();
});
