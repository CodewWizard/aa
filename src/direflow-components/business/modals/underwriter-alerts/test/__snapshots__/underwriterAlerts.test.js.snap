// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<a
  className="btnStyle btnPrim display-block mb-10"
  style={
    Object {
      "textDecoration": "none",
    }
  }
>
  Review UW Alerts
</a>
`;

exports[`renders without crashing without props 1`] = `
<a
  className="btnStyle btnPrim display-block mb-10"
  style={
    Object {
      "textDecoration": "none",
    }
  }
>
  Review UW Alerts
</a>
`;
